src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService serviceInform = new UserService();
+        User user = new User(email, password);
+        boolean info = serviceInform.findByEmail(email)
+                != null && user.getPassword()
+                .equals(password) ? true : false;
+        return info;
```

Comment:
rewrite your solution to not use boolean literals like true or false, use logical expressions directly
also - In situations like this, there is no need to store the object in a local variable if it will be immediately used. Instead, you can pass the expression directly without assigning it to a variable